app/assets/stylesheets/style.scss

```diff
@@ -393,3 +394,74 @@ label:hover {
   }
 }
 
+.side-nav {
+  transition: all 1s ease;
+  background-color: #F2F2F2;
+  top: 40%;
+  left: 0;
+  position: fixed;
+  width: 7em;
+  font-family: 'Raleway', sans-serif;
+  padding: 50px, 0;
+  border-radius: 0 10px 10px 0;
+  z-index: 1;
+
+
+  li {
+
+  }
+
+  .active {
+    background-image: -webkit-gradient(
+    linear,
+    left bottom,
+    right bottom,
+    color-stop(0.02, $accent-blue),
+    color-stop(0.02, $accent-orange)
+    ) !important;
+    background-image: -o-linear-gradient(right, $accent-blue 2%, $accent-orange 2%) !important;
+    background-image: -moz-linear-gradient(right, $accent-blue 2%, $accent-orange 2%) !important;
+    background-image: -webkit-linear-gradient(right, $accent-blue 2%, $accent-orange 2%) !important;
+    background-image: -ms-linear-gradient(right, $accent-blue 2%, $accent-orange 2%) !important;
+    background-image: linear-gradient(to right, $accent-blue 2%, $accent-orange 2%) !important;
+    color: $accent-orange !important;
+  }
+
+}
+
+.side-nav {
+    -webkit-animation: fadein 2s; /* Safari, Chrome and Opera > 12.1 */
+       -moz-animation: fadein 2s; /* Firefox < 16 */
```

Comment:
Use `//` comments everywhere<br>Line should be indented 2 spaces, but was indented 7 spaces<br>Avoid vendor prefixes.